etc/kayobe/kolla.yml

```diff
@@ -231,7 +231,7 @@ kolla_enable_ironic: false
 #kolla_enable_manila_backend_cephfs_nfs:
 #kolla_enable_mariabackup:
 #kolla_enable_mistral:
-#kolla_enable_monasca:
+kolla_enable_monasca: yes
```

Comment:
I think it might be too heavy to enable monasca by default. If we add the necessary configuration but keep the flag as no (default), it will be easy to enable it if required. That can be added to the docs.